### PR TITLE
Corrected and completed the postgis2 migration doc

### PR DESCRIPTION
**Pull request by ochriste**
- correct automatic <geometry_column> name use in constraint
- add comment about VIEWS + helper sql

### CODE REVIEW

File: doc/integrator/upgrade_application.rst [hunk 1]
```diff
@@ -445,11 +445,22 @@ You need to create the ``migrategeomtopostgis2`` function first (simply copy
 and input the function definition above in your terminal), then execute the
 ``select`` (adapted to your need).
 
-If the following constraints dont exist, ``enforce_dims_geom``,
-``enforce_geotype_geom`` or ``enforce_srid_geom``, the query will output some
-``NOTICE``, which may mean you have used other names for your constraints, so you
-should have a look at the corresponding tables and remove the constraints
-manually.
+If the following constraints dont exist, ``enforce_dims_<geometry_column>``,
```
Comment (asaunier) [inline doc/integrator/upgrade_application.rst]: dont exist => do not exist

File: doc/integrator/upgrade_application.rst [hunk 2]
```diff
@@ -445,11 +445,22 @@ You need to create the ``migrategeomtopostgis2`` function first (simply copy
...
+``enforce_geotype_<geometry_column>`` or ``enforce_srid_<geometry_column>``, 
+the query will output some ``NOTICE``, which may mean you have used other
```
Comment (asaunier) [inline doc/integrator/upgrade_application.rst]: mean => means

File: doc/integrator/upgrade_application.rst [hunk 3]
```diff
@@ -445,11 +445,22 @@ You need to create the ``migrategeomtopostgis2`` function first (simply copy
...
+If the following constraints dont exist, ``enforce_dims_<geometry_column>``,
+``enforce_geotype_<geometry_column>`` or ``enforce_srid_<geometry_column>``, 
+the query will output some ``NOTICE``, which may mean you have used other 
+names for your constraints, so you should have a look at the corresponding 
+tables and remove the constraints manually.
+
+If you have created views depending on the modified table, you need to drop 
+and recreate all the related views.
+
+here are some helper queries to generate sql files containing the views DROP and
```
Comment (asaunier) [inline doc/integrator/upgrade_application.rst]: here => Here
sql => SQL

**Comment by sbrunner** [comment]
Pleas start your commit message with a capital, it's nicer :-)

File: doc/integrator/upgrade_application.rst (hunk 2 above)
Comment (elemoine) [inline doc/integrator/upgrade_application.rst]: No. "may mean" is correct :)

Comment (asaunier) [inline doc/integrator/upgrade_application.rst]: Indeed, I have omitted "may".

**Comment by sbrunner** [comment]
Looks good, merge :-)

**Pull request merged**